### PR TITLE
add swagger ui in local cluster

### PR DESCRIPTION
**Pull request by xilabao**
use `ENABLE_SWAGGER_UI=true hack/local-up-cluster.sh`

### CODE REVIEW

**Comment by resouer** [comment]
@k8s-bot ok to test

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -62,6 +62,9 @@ CLOUD_PROVIDER=${CLOUD_PROVIDER:-""}
 CLOUD_CONFIG=${CLOUD_CONFIG:-""}
 FEATURE_GATES=${FEATURE_GATES:-"AllAlpha=true"}
 
+# enable swagger ui
+ENABLE_SWAGGER_UI=${ENABLE_SWAGGER_UI:-false}
+
```
Comment (resouer) [inline hack/local-up-cluster.sh]: May I ask the reason/use case you want to enable `ENABLE_SWAGGER_UI`? Personally I don't think it's a frequently used flag for developers (the main users of this scripts)

Comment (xilabao) [inline hack/local-up-cluster.sh]: the official site (https://kubernetes.io/kubernetes/third_party/swagger-ui/) seems to be unstable. Could we to make a simple way to use swagger-ui?

**Comment by xilabao** [comment]
> [APPROVALNOTIFIER] This PR is NOT APPROVED

@jbeda @resouer @eparis

**Pull request merged**